src/Parsers/PopularBankParser.php

```diff
@@ -31,12 +30,22 @@ public function parse(AbstractFile $file)
                 return;
             }
 
-            $collection->push(new Deposit($line[2], $line[0], $line[5], $line[1]));
+            $collection->push(new Deposit($line[2], $line[0], $line[5], $line[1], $this->uniqueId($line)));
```

Comment:
If every object has a unique id then it is not necessary to pass it via constructor.

```suggestion
$collection->push(new Deposit($line[2], $line[0], $line[5], $line[1]));
```